src/content/community/grants/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Ethereum Foundation & community grant programs
+description: A listing of the grant programs throughout the Ethereum ecosystem.
+lang: en
+---
+
+# Ethereum Grants {#ethereum-grants}
+
+The programs listed below offer a variety of funding grants for projects working to promote the success and growth of the Ethereum ecosystem. Use this as a guide to find and apply for funds to help make your next Ethereum project a success.
+
+### Broader Ecosystem {#broader-ecosystem}
+
+- [EF Ecosystem Support Program](https://esp.ethereum.foundation) - _Rollups, community and education, tooling and testing, infrastructure, interoperability, standards and research, wallets, and more_
+- [Consensys](https://consensys.net/grants/) - _Infrastructure L1 and L2, education & technical knowledge, usability and dev tooling, social impact, security, and more_
+- [MetaCartel](https://www.metacartel.org/) - _Dapp development_
+- [Moloch DAO](https://www.molochdao.com/) - _Privacy, layer 2 scaling, client security, and more_
+- [Open Grants](https://opengrants.com/explore)
+
+### Project Oriented {#project-oriented}
+
+- [Maker](https://grants.makerdao.com/) - _Risk, governance, core development, ecosystem, tools, MarerDAO community, security_
+- [TheGraph](https://airtable.com/shreX09LazIhsg0bU)
+- [Uniswap](https://airtable.com/shrEXXxXB1humz7VS)
+
+### Quadratic Funding {#quadratic-funding}
+
```

Comment:
I think we need at least a one-line explainer here. Take something from the [DeFi](https://ethereum.org/en/defi/#quadratic-funding) explanation of it.